RestClient.cpp

```diff
@@ -238,22 +226,16 @@ int RestClient::readResponse(String* response) {
     int i = 0;
     int code = 0;
 
-    if(response == NULL){
-        HTTP_DEBUG_PRINT("HTTP: NULL RESPONSE POINTER: \n");
-    }else{
-        HTTP_DEBUG_PRINT("HTTP: NON-NULL RESPONSE POINTER: \n");
-    }
-
-    HTTP_DEBUG_PRINT("HTTP: RESPONSE: \n");
     void* http_client;
     if(ssl) {
         HTTP_DEBUG_PRINT("HTTP: Connect: " + String(sslClient.connected()) + " Available: " + String(sslClient.available()) + "\n");
         while (sslClient.connected()) {
-            HTTP_DEBUG_PRINT(".");
+            // HTTP_DEBUG_PRINT(".");
+            delay(0);
```

Comment:
Why add this delay?  Probably should remove the comment on `HTTP_DEBUG_PRINT` as well - lets keep the original debug output consistent.